lib/cn/cndb.c

```diff
@@ -2194,6 +2194,7 @@ cndb_cn_count(struct cndb *cndb, u32 *cnt)
     return 0;
 }
 
+/* [HSE_REVISIT] tstate used to use cndb_cn_blob_get().  does anybody else?  can we lose it? */
```

Comment:
Can we just do a grep and see if anybody else is calling this? What would be required to figure this out?

Reply:
I think we hand that off to @gsramdasi  since he's deep into CNDB at this point.

Reply:
Yes, I've removed all references to this function